tests/test_tasks/test_clustering_task.py

```diff
@@ -1,46 +1,29 @@
 import openml
-
 from .test_task import OpenMLTaskTest
 
 
 class OpenMLClusteringTaskTest(OpenMLTaskTest):
 
-    def setUp(self):
+    __test__ = True
+
+    def setUp(self, n_levels: int = 1):
 
         super(OpenMLClusteringTaskTest, self).setUp()
-        # no clustering tasks on test server
-        self.production_server = 'https://openml.org/api/v1/xml'
-        self.test_server = 'https://test.openml.org/api/v1/xml'
-        openml.config.server = self.production_server
         self.task_id = 146714
+        self.task_type_id = 5
         self.estimation_procedure = 17
+        self.production_server = "https://openml.org/api/v1/xml"
```

Comment:
Isn't that available in the main testing class?

Reply:
Yes, it is. Fixed.